QA.md

```diff
@@ -1,4 +1,4 @@
-# Windows AI Studio Q&A
+# AI Toolkit Q&A
 
 ### Do I need to fine-tune to test Phi-2?
```

Comment:
It is out of date, users now can test models in playground without finetunning them